twilight-cache-inmemory/src/event/emoji.rs

```diff
@@ -9,7 +9,7 @@ use twilight_model::{
 impl InMemoryCache {
     pub(crate) fn cache_emojis(&self, guild_id: Id<GuildMarker>, emojis: Vec<Emoji>) {
         if let Some(mut guild_emojis) = self.guild_emojis.get_mut(&guild_id) {
-            let incoming: Vec<_> = emojis.iter().map(|e| e.id).collect();
+            let incoming: Vec<_> = emojis.iter().map(|e| e.id.unwrap()).collect();
```

Comment:
I think it's better to filter emojis without an `Id` out here with `emojis.iter().filter_map(|e| e.id).collect();`, since those emoji should probably not be cached anyway